swagger_parser/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 1.6.4
+- Add new config parameter `mark_files_as_generated`
```

Comment:
Add version 1.7.0 for this and include all other fixes

Reply:
Edited in https://github.com/Carapacik/swagger_parser/pull/90/commits/b4ff6e95e2ff5a67979bd72fd4951282b67df4ad